core/chains/cosmos/config.go

```diff
@@ -17,37 +17,37 @@ import (
 
 	"github.com/smartcontractkit/chainlink/v2/core/chains"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/cosmos/types"
-	v2 "github.com/smartcontractkit/chainlink/v2/core/config/v2"
+	"github.com/smartcontractkit/chainlink/v2/core/utils/configutils"
```

Comment:
nit/ I think it would be more idiomatic to name the package just `config` to avoid repeating `utils`:
```suggestion
	"github.com/smartcontractkit/chainlink/v2/core/utils/config"
```
Can still name the imports when there are collisions :shrug: 
```suggestion
	configutils "github.com/smartcontractkit/chainlink/v2/core/utils/config"
```